### PR TITLE
Update 1907-02-25.xml

### PR DESCRIPTION
**Pull request by ashleyrc23**
Working on page 2 and 3

### CODE REVIEW

**Comment by huoyanzhizhou** [comment]
Please put your page 2 content into the div type="page" n="2" and page 3 content into the div type="page" n="3", for now your page "div"s are empty except your page 1 div.  Also, you have cb but you did not point out which column you looking at (like cb n="6").

**Comment by whanley** [comment]
Hi Ashley, looks like you are doing lots of good work here. But I don't know which file you want to submit. There are two in this pull request. Make sure that the best material you have is in 1807-02-25.xml, then delete the file called 1907-02-25 copy.xml and send a new pull request.

**Comment by whanley** [comment]
Good, thanks. Now there is a structure problem to address: you are starting new divs before you close the previous div. That's why you have about 10 `</div>`s in a row at the end of your file.

The first problem like this is on line 1743. Use the outline to find the divs that are not closed.

Otherwise the document looks really quite good. Lots of info to update in ads and templates, of course.

Also, make sure to use the features attribute on divs were appropriate. More info here: https://dig-eg-gaz.github.io/contents/features/

**Comment by whanley** [comment]
This is fabulous work. These issues are almost complete. Each needs a bit of tidying--divs need to be filled out, too much bold in some issues, some ads and templates to be updated. But mostly very, very good. If you want to sit down in office hours for 20 minutes I can help you put the finishing touches on them.

File: 1907-03-01.xml [hunk 1]
```diff
@@ -0,0 +1,6480 @@
...
+                            <!-- Enter issue number -->
+                            <cell> ALEXANDRIA, FRIDAY MARCH 1, 1907.</cell>
+                            <!-- Enter date -->
+                            <cell> [EIGHT PAGES</cell>
+                            <!-- Enter number of pages -->
+                            <cell> P.T. 1</cell>
+                        </row>
+                    </table>
+                </div>
+                <cols n="3"/>
```
Comment (huoyanzhizhou) [inline 1907-03-01.xml]: Remember to use cols. I locate many cb in your file, but only one cols.

Comment (huoyanzhizhou) [reply 1907-03-01.xml]: Also, remember to update your measures.

**Comment by huoyanzhizhou** [comment]
The files are overall good. But your new file of 1907-03-02 still only got one cols just like 03-01 page, and there are still measures need to be updated. Are you using the same template for every file? Since there are many files in this pull request with overall good quality. I would let @whanley to decide if it should be merged. Anyway, please keep your good work and updating your files!

**Comment by whanley** [comment]
Yes, I'll merge but please keep tidying!

**Pull request merged**